vhdbuilder/packer/install-dependencies.sh

```diff
@@ -269,28 +269,32 @@ done
 # NOTE that we keep multiple files per k8s patch version as kubeproxy version is decided by CCP.
 KUBE_PROXY_IMAGE_VERSIONS="
 1.17.13
-1.17.13-hotfix.20210310.1
+1.17.13-hotfix.20210310.2
 1.17.16
-1.17.16-hotfix.20210310.1
+1.17.16-hotfix.20210310.2
 1.18.8-hotfix.20200924
-1.18.8-hotfix.20201112.1
+1.18.8-hotfix.20201112.2
 1.18.10-hotfix.20210118
-1.18.10-hotfix.20210310.1
+1.18.10-hotfix.20210310.2
 1.18.14-hotfix.20210118
 1.18.14-hotfix.20210322
+1.18.14-hotfix.20210322.1
 1.18.17-hotfix.20210322
+1.18.17-hotfix.20210322.2
 1.19.1-hotfix.20200923
 1.19.1-hotfix.20200923.1
 1.19.3
 1.19.6-hotfix.20210118
 1.19.6-hotfix.20210310.1
 1.19.7-hotfix.20210310
-1.19.7-hotfix.20210310.1
+1.19.7-hotfix.20210310.2
```

Comment:
will your pipeline automatically update the kubeproxy image version in RP side for this change?

Reply:
yes, the sequence is auto patch pr in rp.
and I will "copy" the changes in rp here.